en/gps_compass/rtk_gps_locosys_r2.md

```diff
@@ -0,0 +1,112 @@
+# Locosys RTK-compass GPS

```

Comment:
This is already in docs - remove http://docs.px4.io/main/en/gps_compass/rtk_gps_locosys_r2.html